trusted-committer-article.md

```diff
@@ -490,21 +465,22 @@ contribution to the community and accelerate your own growth.
 
 In the past sections, we have learned about the responsibilities of TCs;
 ensuring product quality, keeping their community healthy, reducing the barrier
-to making contributions as well as upleveling the community and advocating the
-communities needs in their organization. We also talked about how to become a
-TC and what it takes to fill that role. For me personally, I can say that my
-years working as TC have been the best years of my professional life, so far.
-It's been a great experience, overall. In that sense, I hope this this video
-inspired you to set off on a path towards becoming a TC as well. And if you're
-doubt, why not just try it out?
-
-We also hope that this video will help your organization understand
-the importance of having capable TCs for the success of any InnerSource 
-initiative and the level of empowerment that this role requires.
+to making contributions, upleveling the community and advocating the
+community's needs in their organization. We also talked about how to become a
+TC and what it takes to fill that role. Working as a TC will be demanding but
+it will also be very fulfilling and will help you amplify your value contribution
+in your company.
+
+In that sense, we hope that this video inspired you to set off on a path
+towards becoming a TC as well. And if you're doubt, why not just try it out?
```

Comment:
> And if you're doubt, why not just try it out?

I think that this line works out well in the spoken text, but feels a little off-tone here in writing.  Recommend just removing it.